new-lamassu-admin/src/components/buttons/Link.js

```diff
@@ -6,23 +6,25 @@ import styles from './Link.styles'
 
 const useStyles = makeStyles(styles)
 
-const Link = memo(({ submit, className, children, color, size, ...props }) => {
-  const classes = useStyles()
-  const classNames = {
-    [classes.link]: true,
-    [classes.primary]: color === 'primary',
-    [classes.secondary]: color === 'secondary',
-    [classes.noColor]: color === 'noColor'
-  }
+const Link = memo(
+  ({ submit, className, children, color = 'primary', ...props }) => {
+    const classes = useStyles()
+    const classNames = {
+      [classes.link]: true,
+      [classes.primary]: color === 'primary',
+      [classes.secondary]: color === 'secondary',
+      [classes.noColor]: color === 'noColor'
+    }
```

Comment:
ooops, pressed wrong button on the review screen and had not noticed my comment/question was on pending state.

but yes, it looks goof for me

Reply:
@mautematico this can be done either like this or passing the variable through useStyles, like shown here: https://github.com/lamassu/lamassu-server/blob/bda29ad150354b1915e298ec3972f8ac06023860/new-lamassu-admin/src/components/buttons/Button.js#L10